intellij/src/saros/intellij/ui/messages.properties

```diff
@@ -183,9 +183,11 @@ ModuleTab_module_base_path_file_chooser_title=Choose Module Base Path
 ModuleTab_module_base_path_file_chooser_description=Choose the base bath used for the new module.
 ModuleTab_use_existing_module=Use existing module
 ModuleTab_use_existing_module_local_module=Local module
-ModuleTab_create_new_module_name_invalid_tooltip=The module name must not contain separator characters and must not match any existing module name for the chosen project.
-ModuleTab_create_new_module_base_path_invalid_tooltip=The given module base path does not exist or does not point to a directory.
-ModuleTab_use_existing_module_local_module_invalid_tooltip=An existing module must be chosen.
+ModuleTab_create_new_module_name_invalid_tooltip=The directory name must not be empty and must not contain separator characters.
+ModuleTab_create_new_module_base_path_invalid_tooltip=An existing directory that is part of the project content must be chosen.
+ModuleTab_existing_directory_path_file_chooser_title=Choose Existing Directory
+ModuleTab_existing_directory_path_file_chooser_description=Choose an existing directory to represent the shared reference point.
```

Comment:
Do we want to communicate the concept "Reference Point" to users?
I think this is a general question. I would expect that the user has the perception "This is the directory I want to share" and I think we should talk with the users in their language.

Reply:
But we still need a way to communicate to the user that certain directories are sharing roots and others aren't. For example, deleting a shared inner directory is fine with Saros. Deleting a reference point/sharing root directory, however, can not be handled by Saros and will lead to the session breaking down.

We could change the wording to "root directory" or "sharing root directory", but a similar naming scheme/concept is already used in a by IntelliJ IDEA with its content/source/resource roots, so we would have to be careful not to cause confusion.

Reply:
True, I think "root directory" is more intuitive.

Reply:
Resolved in #1008.